frontend/new-admin-vm/Install-CAM.ps1

```diff
@@ -210,6 +222,48 @@ Configuration InstallCAM
 			MatchSource = $false
 		}
 
+		xRemoteFile Download_Install_Agent.ps1
```

Comment:
When do we use the 'naked' ps1 file? Maybe I'll see later.

Reply:
at the moment we don't, my understanding was the we may potentially allow customers to customize this and rebuild zip file since its in their blob. If this is not correct, then i will remove it.

Reply:
No real harm in copying I guess. Just making sure I knew where it was used :)